serverless/messaging/how-to/migrate-to-new-mnq.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: How to get migrate to the new Messaging and Queuing version
+  description: This page explains how to migrate to the new Messaging and Queuing version
+content:
+  h1: How to migrate to the new Messaging and Queuing version
+  paragraph: This page explains how to migrate to the new Messaging and Queuing version
+tags: messaging queuing enable disable migrate migration v1alpha1 v1beta1 nats sqs sns nats-account account create 
+categories:
+  - serverless 
+dates:
+  validation: 2023-10-03
+  posted: 2023-10-03
+---
+
+This page shows how to migrate to the new version of Scaleway Messaging and Queuing that was recently released. The migration process depends on the messaging protocols you use. If you were not using Scaleway Messaging and Queuing before 16 October 2023, this document does not concern you.
+
+For an in-depth explanation of the migration process, billing and General Availability, see our [dedicated reference documentation](/serverless/messaging/reference-content/migration/).
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You were using Scaleway Messaging and Queuing before 16 October 2023
+</Message>
+
+## NATS
+
+NATS namespaces and credentials created with Scaleway Messaging and Queuing before October 16 2023 **have been automatically migrated to the new version**.
+
+1. Log in to the [Scaleway console](https://console.scaleway.com/) and click **Messaging** in the side menu, then click **NATS**. You see that your NATS namespaces have now become [NATS accounts](/serverless/messaging/concepts/#nats-account). The credentials of your namespaces are now credentials of your NATS accounts.
+
+2. **Update your endpoints.** The endpoint of your Scaleway Messaging and Queuing NATS service has changed. The new endpoint is displayed in the console. **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.
```

Comment:
maybe add this part to a message box? ` **You must update the endpoint in any code or applications using these endpoints.** After 30 days, the old endpoints will no longer be supported.`